### DISCUSSION TITLE
Help Request: I am new in contribution. i am trying to set the the project in my local and i successfully set up the front end part. But i am not able to set up the back end part in the project.

### DISCUSSION THREAD

**Discussion by kashyapshukla** [TEAMMATES/teammates#12651] [Help requests]
### Environment

I am new to contribution.

how to set up the backend part
- **Operating System: Windows
- **JDK Version:17

your guidance will be appreciated

thank you.

### Intention

I am trying to make my first contribution

**Comment by weiquu** [comment]
Hi @kashyapshukla, have you tried following the steps in the Development section of the documentation [here](https://teammates.github.io/teammates/development.html#managing-the-dev-server-back-end)?
